### PR TITLE
added multisearch

### PR DESCRIPTION
**Pull request by uberhyper555**
You can search for multiple pokemon by separating search terms with a ";"
eg. "pidgey;weedle" will show all pidgeys and all weedles.

### CODE REVIEW

**Comment by Ljaysoft** [comment]
it's outputting weird stuff in the console, this change needs to be tested/debuged

**Comment by uberhyper555** [comment]
What is being printed to the console?

File: src/me/corriekay/pokegoutil/windows/PokemonTab.java [hunk 1]
```diff
@@ -594,14 +594,18 @@ private JPanel _buildPanelForOperation(String operation, ArrayList<Pokemon> poke
...
                 String searchme = PokeHandler.getLocalPokeName(poke) + "" + poke.getPokemonFamily() + poke.getNickname()
                         + poke.getMeta().getType1() + poke.getMeta().getType2() + poke.getMove1() + poke.getMove2()
                         + poke.getPokeball();
                 searchme = searchme.replaceAll("_FAST", "").replaceAll("FAMILY_", "").replaceAll("NONE", "").replaceAll("ITEM_", "").replaceAll("_", "").replaceAll(" ", "").toLowerCase();
-                if (searchme.contains(search)) {
-                    pokes.add(poke);
+                for(String s:terms) {
+                    if (searchme.contains(s)) {
+                        pokes.add(poke);
+                        break;
```
Comment (Wolfsblvt) [inline src/me/corriekay/pokegoutil/windows/PokemonTab.java:607]: Why is here a `break` statement? Does it do something? :D

**Comment by uberhyper555** [comment]
The break is so that the same pokemon doesn't get added more than once. for example pidgeotto;pidgey pidgeottos searchme string has pidgeotto and pidgey in it so it would get added twice. I could replace the break with a boolean flag if that's better.

**Comment by Wolfsblvt** [comment]
Nah, it's fine. Had a thought problem, but got it now.
Maybe just make a line comment above that explains the break. Should be enough.

Oh, an if @Ljaysoft doesn't find the time to check the errors, I wil review your PR tomorrow and test locally. Just don't have the time right now.

**Comment by Wolfsblvt** [comment]
Works like a charm!
Have tested it locally. I'll add the comment myself in the next refactoring.

**Pull request merged**